<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PwDialog</class>
 <widget class="QDialog" name="PwDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>208</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans Mono CJK TC</family>
    <pointsize>11</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Egge's Messenger: Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>9</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 9pt &quot;Noto Sans&quot;;</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <widget class="QWidget" name="tabLogin">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>9</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="accessibleName">
       <string>  Login  </string>
      </property>
      <attribute name="title">
       <string>  Login  </string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBoxGloballyVisible">
         <property name="toolTip">
          <string>User visible to all other users</string>
         </property>
         <property name="text">
          <string>Globally visible</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>User:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditUser1"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditPw1">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRegister">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="accessibleName">
       <string>New user</string>
      </property>
      <attribute name="title">
       <string>  Register  </string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>User:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditUser2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPw2">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Repeat password:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditPwRepeat2">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabScorchedEarth">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>9</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Scorched earth means you are leaving nothing behind. You burn everything. This is the ultimate precaution, the ultimate get-away.</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <attribute name="title">
       <string>  Precaution  </string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Loggin in with this password leaves schorched earth behind.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Repeat password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditPw3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditPw3Repeat">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PwDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PwDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
